### ISSUE TITLE
[Accessibility Docs] Tooltips

### ISSUE THREAD

**Issue by sowasred2012** [canonical-web-and-design/vanilla-framework#4068]
Using [this template](https://docs.google.com/document/d/1IwFMfJrTfJPXDcVJfLWXYkw-pjgWhBWo4GX8X098Q64/edit#), create accessibility docs for [Tooltips](https://vanillframework.io/docs/patterns/tooltips)

**Comment by bethcollins92** [comment]
This [blog](https://sarahmhigley.com/writing/tooltips-in-wcag-21/) mentions how when the tooltip isn't active we should be using aria-hidden="true" on the descriptive text. 

I tried using voice over on the tooltips, it's not great. the description is read out after the label text of the button, and then read out again straight after. I tried to add aria-hidden to the description but this hides it both times. Could we use aria-labelledby instead?

**Comment by bethcollins92** [comment]
Here is the first draft of the tooltip [accessibility doc](https://docs.google.com/document/d/1Pgk4JYHIIbdkFmAzrPBKO1uNQnzmAeXqwTKtoiajoCA/edit#)

**Comment by bethcollins92** [comment]
Notes are up here: https://vanillaframework.io/docs/patterns/tooltips#accessibility
